### ISSUE TITLE
Question. Supply gas from the contract instead of the message sender sending it

### ISSUE THREAD

**Issue by subramanianv** [blockchaincan/beer-machine#3]
Is there a way for the smart contract to supply the gas instead of the message sender sending it.

I am trying to build a registry using uport-persona. For creating a persona, the user has to send some ether for the gas. Is there a way around this where the gas is supplied from the proxy contract ?

**Comment by featheredape** [comment]
No, but I think that feature might be added in a future version of Ethereum. For now, gas costs are paid msg.sender
